composer.json

```diff
@@ -14,13 +14,19 @@
     ],
     "require": {
         "php": ">=5.4",
-        "league/commonmark": "0.11.*"
+        "league/commonmark": ">=0.11,<0.19"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^7.1"
```

Comment:
that version of phpunit is not php 5.4 compatible. altho for tests probably ok to have higher php requirement, even it seems super odd.